tests/e2e/specs/wordpress/embedBlock.js

```diff
@@ -81,7 +81,7 @@ describe('Embed Block', () => {
     await insertBlock('Web Stories');
 
     await page.type(
-      'input[aria-label="Web Story URL"]',
+      'input[aria-label="Story URL"]',
```

Comment:
Again, why has this changed?

Reply:
As answered above,
https://github.com/rtCamp/web-stories-wp/pull/150#discussion_r543569065

Reply:
Yes this change looks correct to me.